Add changes in retrospect (#48)
sw-bachelor/sw6: report/sections/sprint1/sprint-1-retrospective.tex

These new groups started to discus the processes and talked about the shortcomings of the sprint.
This resulted in each of these new groups coming up with some new ideas of what could be done differently.
When everyone was done discussing, all these new ideas were put up on a website where each person could vote anonymously on the three ideas they liked best.
\\\\
The ideas presented have been compiled into a list as shown below:
\begin{itemize}
    \item Not everyone needs to be at the sprint planning.
    \item Remember to come ask the PO group if in doubt about anything related to the user story.
    \item Less meetings on days without lectures.
    \item Get some of the developers to do technical explanations of the user stories so that there is a customer and a developer perspective on each user story.
    \item More systematic standup meetings. It should be enforced that every group has a representative present.
    \item Stop having sprint planning meetings.

>     \item Use the F-club's facilities as a more relaxed space for discussion. 

Reviewer: ```suggestion
    \item Use F-Klubben's facilities as a more relaxed space for discussion. 
```

    \item Standup meetings should not be placed in the middle of \textit{Advanced Algorithms} exercise sessions.
    \item Acquire new tablets since some of the current ones are in bad condition or are too old to run the newest versions of their operating system.
    \item Consider doing online standup meetings or allow people to attend the meetings virtually.
    \item Less meetings on days with lectures.
    \item Standup meetings should not be right after or before a lecture.

>     \item More hack-a-thons.

Reviewer: ```suggestion
    \item More hackathons.
```

    \item Consider when the PO group should review functionality. 
    \item The PO group should go around and ask how far the different groups are with their tasks.
    \item More activities across the different groups.
\end{itemize}
Afterwards the sprint retrospective was over.
The process group then took the ideas and started considering how they could incorporate them into the next sprint.
\\\\
The idea that each user story should have an technical explanation was the idea with the most votes. 
Following the meeting it was quickly introduced as a new standard for new user stories and existing user stories got a technical explanation added.
Furthermore, development groups were encouraged to come see the PO group more in case of doubt which seemed to have worked.
The process group decided that the sprint planning meeting should be changed to the sprint introduction.

> These processes are described in \ref{subsec:SoS-sprint-planning} and \ref{subsec:SoS-sprint-introduction} respectively.

Reviewer: ```suggestion
These processes are described in \autoref{subsec:SoS-sprint-planning} and \autoref{subsec:SoS-sprint-introduction} respectively.
```
